src/Common/mysqlxx/mysqlxx/Pool.h

```diff
@@ -64,17 +64,6 @@ class Pool final
             decrementRefCount();
         }
 
-        Entry & operator= (const Entry & src) /// NOLINT
```

Comment:
This assignment operator may be error-prone, and it is possible for use-after-free, it is not possible in the current usage, but still, imagine that before `incrementRefCount`, `mysqlxx::Pool::tryGet` removes this connection due to `ref_count == 0` - https://github.com/ClickHouse/ClickHouse/blob/76119a4567ce2ac9c0aff715c1a9ba2607e806e0/src/Common/mysqlxx/Pool.cpp#L232-L244